src/CommunityToolkit/Xamarin.CommunityToolkit/Views/BaseTemplatedView.shared.cs

```diff
@@ -1,14 +1,12 @@
-﻿using System.ComponentModel;
-using Xamarin.Forms;
+﻿using Xamarin.Forms;
 
-namespace Xamarin.CommunityToolkit.UI.Views
+namespace Xamarin.CommunityToolkit.UI.Views.Internals
 {
-	[EditorBrowsable(EditorBrowsableState.Never)]
 	public abstract class BaseTemplatedView<TControl> : TemplatedView where TControl : View, new()
 	{
 		protected TControl Control { get; private set; }
 
-		public BaseTemplatedView()
+		internal BaseTemplatedView()
```

Comment:
Maybe let's preserve it public (e.g. somebody decides to build own view. Not sure...)